android/app/project.properties

```diff
@@ -0,0 +1,14 @@
+# This file is automatically generated by Android Tools.
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file must be checked in Version Control Systems.
+#
+# To customize properties used by the Ant build system edit
+# "ant.properties", and override values to adapt the script to your
+# project structure.
+#
+# To enable ProGuard to shrink and obfuscate your code, uncomment this (available properties: sdk.dir, user.home):
+#proguard.config=${sdk.dir}/tools/proguard/proguard-android.txt:proguard-project.txt
+
+# Project target.
+target=android-21
```

Comment:
That's the project build target and it's recommended to use the latest Android version.
There are ways to check the build inside the code for hanlding older version's code.

The versions you mean are minSdkVersion and targetSdkVersion, which are already defined in AndroidManifest.xml